HW04.Rmd

```diff
@@ -77,45 +175,217 @@ Assume that:
 
 ### Are whites who are killed by guns more likely to die because of suicide or homicide? How does this compare to blacks and hispanics?
 
-```{r race}
+**Answer**
+Looking at the gross numbers, some general trends start to arise. It is clear by the data that white people gun deaths are more related to suicide than homicide. On the other hand, black people gun deaths are the opposite, with more deaths related to homicide than suicide. Meanwhile, hispanic population seems about the same in proportion.
+
+```{r race, echo=F}
+
+deaths_race <- guns %>%
+    group_by(race, intent) %>%
+    filter(intent %in% c("Homicide", "Suicide"), 
+           race %in% c("White", "Hispanic", "Black")) %>%
+    count(race) %>%
+    rename(deaths = n) %>%
+    ungroup() #It is important to ungroup to perform further analysis
+
+#A simple graph comparing the total number of gun deaths per race and intent.
+deaths_race %>%
+    ggplot(aes(x = race, y = deaths, fill = intent)) +
+    geom_col(position = "dodge") +
+    labs(title = "Total number of gun deaths per race and intent from 2012-2014",
+         x = "Race",
+         y = "Number of gun deaths",
+         caption = "Source: fivethirtyeight.com")
 
 ```
 
+**Answer (continuation)**
+When analyzing percentages per race, it is worth noting that the percentage of black people killed by homicide is around the same as white people that committed suicide (85%). This means that white people is as likely to die from suicide than black people is to die from homicide. For the hispanic population, the likeliness of dying because of homicide is 20% lower than black people, but it remains as the most likely cause of gun death.
+
+
 (hint maybe looking at percentages would be good)
-```{r race_2}
+```{r race_2, echo=F}
+
+#If there's better way of doing the following I don't know how, but I decomposed the deaths_race data frame into races to calculate the percentages of gun deaths by intent individually.

```

Comment:
I'm also not sure if there is a more concise way of doing this, but your workflow is simple and easy to understand! So, I wouldn't worry about it much. I also like that you didn't offset your bar graph this time, and went with coloring based on percentage - clear presentation of data :)